### PR TITLE
switch events to contenteditable

### PR DESCRIPTION
**Pull request by sorax**

### CODE REVIEW

File: assets/js/hooks/events/handler.ts [hunk 1]
```diff
@@ -2,39 +2,40 @@ import { NodeData, UserAction } from "../types";
...
+  setContent,
+  focusNode,
   addEditingUserLabel,
   removeEditingUserLabel,
 } from "../node";
 
 export function handleBlur({ uuid, user_name }: UserAction) {
-  const node = getNodeById(uuid)!;
-  removeEditingUserLabel(node, user_name);
+  // const node = getNodeById(uuid)!;
```
Comment (electronicbites) [inline assets/js/hooks/events/handler.ts:13]: why not delete?

File: lib/radiator_web/live/outline_component.ex [hunk 2]
```diff
@@ -154,128 +144,107 @@ defmodule RadiatorWeb.OutlineComponent do
...
+        "#{prev_node.content}#{content}",
+        user_id,
+        generate_event_id(socket.id)
+      )
+
+      Dispatch.delete_node(uuid, user_id, generate_event_id(socket.id))
+    end
 
     socket
+    # |> stream_delete_by_dom_id(:nodes, "nodes-form-#{uuid}")
```
Comment (electronicbites) [inline lib/radiator_web/live/outline_component.ex:191]: can be deleted as well

File: lib/radiator_web/live/outline_component.ex [hunk 3]
```diff
@@ -154,128 +144,107 @@ defmodule RadiatorWeb.OutlineComponent do
...
+        generate_event_id(socket.id)
+      )
+
+      Dispatch.delete_node(next_node.uuid, user_id, generate_event_id(socket.id))
+    end
 
     socket
-    |> stream_delete_by_dom_id(:nodes, "nodes-form-#{uuid}")
+    # |> stream_delete_by_dom_id(:nodes, "nodes-form-#{uuid}")
     |> reply(:noreply)
```
Comment (electronicbites) [inline lib/radiator_web/live/outline_component.ex:213]: delete ?!

**Review by electronicbites — APPROVED** [review]
lg2m

except very small comments about code which could be deleted

File: assets/js/hooks/events/handler.ts (hunk 1 above)
Comment (sorax) [reply assets/js/hooks/events/handler.ts:13]: in the line below (line 14) we call: `removeEditingUserLabel(user_name);`
this function does remove the user-label (if exists):
```
removeEditingUserLabel(user_name: string) {
  const span = document.getElementById(user_name);
  span && span.remove();
}
```

File: lib/radiator_web/live/outline_component.ex (hunk 2 above)
Comment (sorax) [reply lib/radiator_web/live/outline_component.ex]: ✅ absolutely correct. code is removed now.

File: lib/radiator_web/live/outline_component.ex (hunk 3 above)
Comment (sorax) [reply lib/radiator_web/live/outline_component.ex:211]: ✅ absolutely correct. code is removed now.

**Pull request merged**